pytests/helpers/session.py

```diff
@@ -178,10 +184,41 @@ def prepare_to_run(self):
                 argv += ['-c', path_or_code]
         else:
             pytest.fail()
+        return argv
+
+    def prepare_to_run(self):
+        """Spawns ptvsd using the configured method, telling it to execute the
+        provided Python file, module, or code, and establishes a message channel
+        to it.
+
+        If use_backchannel is True, calls self.setup_backchannel() before returning.
+
+        If perform_handshake is True, calls self.handshake() before returning.
+        """
+        print('Preparing debug session with method %r' % str(self.start_method))
+        argv = []
+        if self.start_method == START_METHOD_LAUNCH:
+            self._listen()
+            argv += self._get_argv_for_launch()
+        elif self.start_method == START_METHOD_CMDLINE:
+            argv += self._get_argv_for_attach_using_cmdline()
+        elif self.start_method == START_METHOD_IMPORT:
+            argv += self._get_argv_for_attach_using_import()
+            ptvsd_path = os.path.dirname(os.path.dirname(ptvsd.__main__.__file__))
```

Comment:
Can you add a TODO comment here saying that we need to get rid of this after we fully switch to Tox? With Tox, ptvsd will be installed in the test environment as a proper package, so no PYTHONPATH patching should ever be necessary).